bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/report/job/AnalyzeJob.java

```diff
@@ -39,78 +46,99 @@ private AnalyzeJob(String name) {
      * @param project The eclipse project that's will be analyzed.
      */
     public AnalyzeJob(IProject project) {
+        this(project, null);
+    }
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641620)